File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.py
class Solution(object):
    def minimumPushes(self, word):
        i=1
        ans=0
        n=len(word)
        while(n>=8):
            n-=8
            ans+=(8*i)
            i+=1
        ans+=(n*i)
        return ans

        

File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.java
class Solution {
    public int minimumPushes(String word) {
       int i=1;
       int ans=0;
       int n=word.length();
       while(n>=8){
        n-=8;
        ans+=(8*i);
        i++;
       }
       ans+=(n*i);
       return ans;
    }
}